mobify/astro-scaffold
# Update scaffold to accommodate the tab navigation for android

Setup Tab navigation for Android

JIRA: https://mobify.atlassian.net/browse/HYB-553
Linked PRs: https://github.com/mobify/astro/pull/539

## Changes
- Add default icon for tab navigation for Android
- Update icon file path so it conforms to Android's standard for assets

## How to test-drive this PR
_Update: the below is no longer relevant as HYB-553-android-tab-bar is currently fully merged in Astro_

This PR is relying on the Astro's HYB-553-android-tab-bar branch
```
git clone git@github.com:mobify/astro.git
git checkout HYB-553-android-tab-bar
cd astro
npm link
```

Then navigate back to your project root directory and run: `npm link astro-sdk` to use your locally developed version of Astro.

## TODOS:
